### ISSUE TITLE
No audio 

### ISSUE THREAD

**Issue by shoully** [foxtrotdev/learn-digits#10]
Hi,

I tried the four versions, but it turn out there's no audio file and the instructions didn't show up!

I'm using Android 10 on Samsung Galaxy J4.

![Screenshot_20231013-091251_Digits](https://github.com/foxtrotdev/learn-digits/assets/680153/169ad70a-b7a9-471e-9d3b-bef5cb82afbd)

**Comment by foxtrotdev** [comment]
1. Do you have any logs? 
1. What language are you using? By default it should use English version. 
1. What about the volume you had set on the phone? Maybe you were on mute?

**Comment by shoully** [comment]
Hello,

I've resolved the issue, the problem arose because I had disabled animations in Developer Mode to improve performance (and I think it's a common). I simply re-enabled animations by adjusting it to .5x instead of OFF from the following:

- Window Animation Scale
- Transition Animation Scale
- Animator Duration Scale

I did the tests on two devices: my physical Samsung J4 and a virtual Pixel 5 (API 29, Android 10). This is how I identified the solution because the Logcat didn't provide any clues.

For your reference, I recorded a video to demonstrate the issue: [No Audio Issue](https://screenpal.com/watch/c06rXpVEqWr)

I hope this helps anyone in the future.
